app/bundles/CampaignBundle/Form/Type/ConfigType.php

```diff
@@ -33,7 +33,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label_attr' => ['class' => 'control-label'],
                 'data'       => $options['data']['campaign_time_wait_on_event_false'],
                 'choices'    => [
-                    'null'  => '0 mn',
+                    'null'  => 'Never retry',
```

Comment:
Thanks for contribution. Make sense for me 👍 
You should use translation: `mautic.core.never`
Also you should rebase it base on 3.x branch, then we can add it to next 3.0.1 release

Reply:
Okay, I will do it, thanks for the advice